rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    book.rental << self
+    person.rental << self
```

Comment:
Because the Book class has rentals and not rental, kindly use `book.rentals` here. This would reflect a has-many relationship since a book has many rentals. The same goes for person.rental which should be `person.rentals`.